src/main/java/httpserver/Resource.java

```diff
@@ -0,0 +1,5 @@
+package httpserver;
+
+public interface Resource {
```

Comment:
I also asked _What is a resource?_

We talked about renaming this to something like `Action`.